src/algebra/category/Module/filtered_colimits.lean

```diff
@@ -83,7 +83,7 @@ instance colimit_has_scalar : has_scalar R M :=
 @[simp]
 lemma colimit_smul_mk_eq (r : R) (x : Σ j, F.obj j) : r • M.mk x = M.mk ⟨x.1, r • x.2⟩ := rfl
 
-instance colimit_module : module R M :=
+instance colimit_distrib_mul_action_with_zero : distrib_mul_action_with_zero R M :=
```

Comment:
```suggestion
-- we provide this separately to the `module` instance below to avoid a timeout
instance colimit_distrib_mul_action_with_zero : distrib_mul_action_with_zero R M :=
```